### PR TITLE
feat: log connect and subscribed events

### PR DESCRIPTION
**Pull request by ethanve**
https://app.asana.com/0/779493871603507/1173922182760674/f

### CODE REVIEW

File: src/AuthorizedSocketConnection.ts [hunk 1]
```diff
@@ -234,6 +235,7 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
       }
     } finally {
       acknowledge(cb);
+      this.log('debug', 'client subscribed');
```
Comment (taion) [inline src/AuthorizedSocketConnection.ts:238]: can you also log details on the actual subscription?

**Comment by taion** [comment]
_maybe_ we want to generate e.g. a UUID or some other ID for the client, so we can sync up "connect" and "disconnect" events too (and link this for the other log events). shrug

File: src/AuthorizedSocketConnection.ts [hunk 2]
```diff
@@ -79,6 +81,7 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
   ) {
     this.socket = socket;
     this.config = config;
+    this.clientId = socket.client.id;
```
Comment (ethanve) [inline src/AuthorizedSocketConnection.ts:84]: @taion I think this is accurate?

Comment (taion) [reply src/AuthorizedSocketConnection.ts:84]: you can simplify slightly i think?

```suggestion
    this.clientId = socket.id;
```

**Pull request merged**